gradle.properties

```diff
@@ -21,7 +21,7 @@ postgresJdbcVersion=42.7.5
 springDocVersion=2.8.5
 
 # For code generation, we run database migrations against an ephemeral database in a Docker
-# container using this image. If you change this, make sure you also change DatabaseTest.kt.
+# container using this image.
 
-postgresDockerRepository=postgis/postgis
-postgresDockerTag=13-3.0
+postgresDockerRepository=terraware/postgres
+postgresDockerTag=v20250227.1
```

Comment:
I'm assuming you're fine with this tag not matching docker-compose?

Reply:
Yes. I'll clarify that in the PR description.